source/as-stream.ts

```diff
@@ -100,6 +100,7 @@ export default function asStream(options: MergedOptions) {
 
 	const pipe = proxy.pipe.bind(proxy);
 	const unpipe = proxy.unpipe.bind(proxy);
+
```

Comment:
Just a nitpick but I prefer to leave no empty lines when hiding original methods and then overridding them. Anyway I'm not sure about it.

@sindresorhus What do you think?

Reply:
I think it's good to have spacing around blocks.